### PR TITLE
docs: added readme.md for Ecom

### PR DESCRIPTION
**Pull request by keshavkumar-in**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,40 @@
...
+</div>
+
+## What's included:
+
+1. [Home Page](https://wrixby-ecom.netlify.app/)
+1. [Product Page](https://wrixby-ecom.netlify.app/product/)
+1. [Cart](https://wrixby-ecom.netlify.app/cart/)
+1. [Wishlist](https://wrixby-ecom.netlify.app/wishlist/)
+1. [Login](https://wrixby-ecom.netlify.app/auth/login.html)
+1. [Sign Up](https://wrixby-ecom.netlify.app/auth/signup.html)
```
Comment (partha8) [inline README.md:20]: the numbering is messed up here as well

**Review by partha8 — COMMENTED** [review]
compress the image, itll help with loading time

File: README.md [hunk 2]
```diff
@@ -1,2 +1,40 @@
...
+
+## Demo
+
+![App Screenshot](https://wrixbyui.netlify.app/assets/images/Wrixby ECom.png)
+
```
Comment (vrinda-mahajan) [inline README.md:25]: Your screenshot link is not working. Please look into it. Everything else looks good.

**Review by vrinda-mahajan — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (keshavkumar-in) [reply README.md:25]: It's added in this same PR, will work after merge

File: README.md (hunk 1 above)
Comment (keshavkumar-in) [reply README.md:20]: Yeah Will correct

**Pull request merged**